### DISCUSSION TITLE
Comer ou não Comer?

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#429] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/comer-ou-nao-comer/

**Comment by angellisy** [comment]
![null](<https://www.vish.blog.br/imagens/comer-ou-nao-comer/images/gallery/11.webp>)

**Comment by brunoaugusto** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/comer-ou-nao-comer/#comment_1142158208>):

> [![null](<https://camo.githubusercontent.com/d6e9597588f94075653a6d4ded275c5e7e62992a7ece7003710b93f5167a0c90/68747470733a2f2f7777772e766973682e626c6f672e62722f696d6167656e732f636f6d65722d6f752d6e616f2d636f6d65722f696d616765732f67616c6c6572792f31312e77656270>)](<https://camo.githubusercontent.com/d6e9597588f94075653a6d4ded275c5e7e62992a7ece7003710b93f5167a0c90/68747470733a2f2f7777772e766973682e626c6f672e62722f696d6167656e732f636f6d65722d6f752d6e616f2d636f6d65722f696d616765732f67616c6c6572792f31312e77656270>)

Qualquer semelhança (não) é mera coincidência:



![null](<https://i.imgur.com/yQy5tEm.png>)
